pkg/codegen/go/gen_program_expressions.go

```diff
@@ -1026,12 +1026,6 @@ func (g *generator) lowerExpression(expr model.Expression, typ model.Type) (
 }
 
 func (g *generator) genNYI(w io.Writer, reason string, vs ...interface{}) {
-	message := fmt.Sprintf("not yet implemented: %s", fmt.Sprintf(reason, vs...))
-	g.diagnostics = append(g.diagnostics, &hcl.Diagnostic{
-		Severity: hcl.DiagError,
```

Comment:
I suppose we could have kept this as a warning?

Reply:
That is also possible, I guess we need to do that for all call-sites of `notImplemented` function calls

Reply:
✅